src/setupProxy.js

```diff
@@ -0,0 +1,35 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+const configUrls = require('../public/config.json');
+
+module.exports = function (app) {
```

Comment:
I feel like we could use a hefty dose of documentation here, and perhaps in `README.md` as well.